src/menu/charprofile.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * This software is licensed under the terms of the MIT License.
+ * See COPYING for further information.
+ * ---
+ * Copyright (c) 2011-2019, Lukas Weber <laochailan@web.de>.
+ * Copyright (c) 2012-2019, Andrei Alexeyev <akari@taisei-project.org>.
+ */
+
+#pragma once
+#include "taisei.h"
+
+#include "menu.h"
+
+void preload_charprofile_menu(void);
```

Comment:
This function is not defined anywhere. I assume you were going to write it, but then added the preload calls to `add_character` instead?

I think it might be better to write this dedicated preload function anyway, and call it in both `create_charprofile_menu`  and when entering the parent "media room" menu. This way the loading delay should be less noticeable (if at all), even if the texture cache is cold and the game has to transcode some of those background textures (e.g. first-time start).